File: sportsbook-app/player-service/src/main/java/com/sportsbook/playerservice/controller/SampleController.java
package com.sportsbook.playerservice.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class SampleController {

    @GetMapping("/api/playerservice/ping")
    public String ping() {
        return "player-service pong";
    }
}

File: sportsbook-app/player-service/src/main/kotlin/com/sportsbook/playerservice/controller/SampleController.kt
package com.sportsbook.playerservice.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class SampleController {

    @GetMapping("/api/playerservice/ping")
    fun ping(): String = "player-service pong"
}
